schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT,
+    name varchar(60),
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL
 );
```

Comment:
- [OPTIONAL] It would be nice to use the `ID` column as the **_primary key_**. The **_primary key_** ensures unique row identification. This results in faster sorting, searching, and querying operations. 👍 😎